chrome_manifest_v3/views/popupView.js

```
@@ -1259,32 +1282,30 @@ Gmail2Trello.PopupView.prototype.clearBoard = function () {
 };
 
 Gmail2Trello.PopupView.prototype.updateBoards = function (tempId = 0) {
-  var self = this;
+  let self = this;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'self' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)